crates/yulgen/src/runtime/functions/revert.rs

```diff
@@ -1,67 +1,46 @@
 use crate::names;
-use crate::names::abi as abi_names;
-use fe_abi::utils as abi_utils;
-use fe_analyzer::namespace::types::{AbiEncoding, FixedSize, Struct, U256};
+use crate::operations::abi as abi_operations;
+use crate::types::{to_abi_selector_names, AbiType};
 use yultsur::*;
 
-fn selector(name: &str, params: &[FixedSize]) -> yul::Expression {
-    let params = params
-        .iter()
-        .map(|param| param.abi_selector_name())
-        .collect::<Vec<String>>();
-
-    literal_expression! {(abi_utils::func_selector(name, &params))}
-}
-
 /// Generate a YUL function to revert with the `Error` signature and the
-/// given set of params.
-/// NOTE: This is currently used for `assert False, "message"` statements which are
```

Comment:
@cburgdorf 

Is there a reason to get rid of this still?

Reply:
I guess that in the future we could just handle that at lowering time by converting `assert false, "message` into `assert false, Error(msg="message")`. This would also mean that the `assert` statements can be used with other types of errors such as `assert false, ApplicationError(code=4711)`